src/help.ts

```diff
@@ -113,4 +114,85 @@ addEventListener("load", async () => {
     if (document.location.hash) {
         document.location.hash = document.location.hash
     }
-})
+}
+
+async function onSettingsPageLoad() {
+    const inputClassName = " TridactylSettingInput "
```

Comment:
CSS on this needs work - on small displays it covers up the setting.

Reply:
Let's just ban users who use Tridactyl on small displays instead.

I didn't know how to fix this with CSS so I just moved the input field after the comments. Might be better like this.